master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -16,74 +14,76 @@ import (
 const (
 	wkspIDQuery       = "workspace_id = ?"
 	wkspIDGlobalQuery = "workspace_id IS ?"
+	// DefaultInvariantConfigStr is the default invariant config val used for tests.
```

Comment:
nit: should these be defined in the test file instead?

Reply:
for some reason putting it the test files makes it unexportable to the api test file, so I put it here